core/domain/acl_decorators_test.py

```diff
@@ -1024,3 +1026,53 @@ def test_exploration_editor_can_access_learner_dashboard(self):
         response = self.testapp.get('/mock/', expect_errors=True)
         self.assertEqual(response.status_int, 200)
         self.logout()
+
+
+class EditTopicDecoratorTest(test_utils.GenericTestBase):
+    """Tests the decorator can_edit_topic."""
+    username1 = 'topicmanager'
+    user_email1 = 'user@example.com'
+    username2 = 'normaluser'
+    user_email2 = 'user2@example.com'
+    topic_id = 'topic_1'
+
+    class MockHandler(base.BaseHandler):
+        @acl_decorators.can_edit_topic
+        def get(self, topic_id):
+            self.render_json({'topic_id': topic_id})
+
+    def setUp(self):
+        super(EditTopicDecoratorTest, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.user_email1, self.username1)
```

Comment:
It's easier to follow things if this is self.manager_email, self.manager_username (and the second one could be viewer_email, viewer_username). Then the reader doesn't have to keep remembering "user 1 is the manager".

Reply:
Done